tools/nl/embeddings/README.md

```diff
@@ -65,6 +65,14 @@ variables.
    ./run.sh -c undata data/curated_input/undata/sheets_svs.csv data/alternatives/undata/*.csv
    ```
 
+   To generate the `bio_ft` embeddings:
+
+   ```bash
+   ./run.sh -c bio data/curated_input/bio/sheets_svs.csv,data/curated_input/main/sheets_svs.csv data/alternatives/main/*.csv
```

Comment:
OK for this PR

But it might be useful to handle the map from index type (`bio`, `undata`, etc) to CSV files in a config/map, rather than list in command line.  Maybe add a TODO if you agree.

Reply:
makes sense, added TODO